cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/TestCommandTest.java

```diff
@@ -227,7 +229,7 @@ public void testTestCommandPreservingBinJarInTargetDir() throws IOException {
         Path projectPath = this.testResources.resolve("validMultiModuleProjectWithTests");
         System.setProperty(ProjectConstants.USER_DIR, projectPath.toString());
         // build the project
-        BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, true , false);
```

Comment:
@hasithaa
This `true` was calling `System.exit()` which for Windows failed the build and for Ubuntu (probably) skipped some tests.